kinto/__init__.py

```diff
@@ -36,6 +36,11 @@
     "group_id_generator": "kinto.views.NameGenerator",
     "record_id_generator": "kinto.views.RelaxedUUID",
     "project_name": "kinto",
+    "mail.host": "localhost",
+    "mail.port": 25,
+    "mail.default_sender": None,
+    "mail.username": None,
+    "mail.password": None,
```

Comment:
This is one way of solving it, but I think it would be better to introduce a sort of registry to be used by plugins. 

kinto-emailer could register expected settings:

```py
config.kinto_setting("mail.host", default="localhost")
config.kinto_setting("mail.port", cast=int, default=25)
config.kinto_setting("mail.password", env="PASSWD")
```

See https://github.com/Kinto/kinto/issues/1544

Reply:
i have no idea how to do this :) is there any plugin as example ?

Reply:
I wrote some details here https://github.com/Kinto/kinto/issues/1544#issuecomment-743241112

Reply:
Let's discuss there ;)